docs/usage.rst

```diff
@@ -387,8 +387,14 @@ accept unsigned content with ``accept_untrusted_content=True``:
     ...                     sender.request_header,
```

Comment:
The sentence above it should be changed as well because it's inaccurate. I think this is better:

In some cases you may not be able to hash request/response content. For example, the content could be too large. If you run into this, Hawk might not be the best fit for you but Hawk does allow you to accept content without a declared hash if you wish.

... important::

  By allowing content without a declared hash, both the sender and receiver are susceptible to content tampering.